lambda/src/main/scala/pricemigrationengine/migrations/DigiSubs2025Migration.scala

```diff
@@ -73,6 +69,35 @@ object DigiSubs2025Migration {
     }
   }
 
+  def brazeName(cohortItem: CohortItem): Option[String] = {
+    /*
+      Canvases:
+          For all USD supporters:
+              SV_DP_PriceRiseUSAnnuals011225
+              c1c16ea3-0290-4bb7-a0cf-57a7ccf174f0
+
+          For non-USD supporters paying monthly:
+              SV_DPPriceRiseUKROWMonthlies011225
+              d2d8f5a3-daec-4ce4-b78c-c367a4736262
+
+          For non-USD supporters paying annually:
+              SV_DP_PriceRiseUKROWAnnuals011225
+              a6dde8ce-0c38-4b00-844a-b10bed1c6a25
+     */
+
+    for {
+      billingPeriod <- cohortItem.billingPeriod
+      currency <- cohortItem.currency
+    } yield {
+      (currency, billingPeriod) match {
+        case ("USD", _)    => "SV_DP_PriceRiseUSAnnuals011225"
```

Comment:
We don't need to split the US subscriptions into annual/monthly?

Reply:
Hmmm, the price and frequency is provided to the canvas so maybe not.

Reply:
Nope :)